cmd/syft/cli/options/catalog.go

```diff
@@ -137,6 +139,8 @@ func (cfg Catalog) ToCatalogerConfig() cataloger.Config {
 		LinuxKernel: kernel.LinuxCatalogerConfig{
 			CatalogModules: cfg.LinuxKernel.CatalogModules,
 		},
+		Java: javaCataloger.NewJavaCatalogerOpts().
+			WithSearchMavenForLicenses(cfg.Java.SearchMavenForLicenses),
```

Comment:
very small nit: it looks like the other lines above and below are just directly constructing the config structs but this is introducing the ".With<thing>()" builder style. since this is just a boolean, and the only property on the struct, it would probably work fine to follow the pattern of the others and not use the builder style

Reply:
Oh my bad - I saw that the Golang cataloger introduced the `With` style and went with that approach.  I'm happy to change it back to a struct with two fields now (Online and URL). I do agree that the `cataloger.Config` should probably be consistent in its style